indexer/services/comlink/src/controllers/api/v4/fills-controller.ts

```diff
@@ -77,6 +88,7 @@ class FillsController extends Controller {
           ? createdBeforeOrAtHeight.toString()
           : undefined,
         createdBeforeOrAt,
+        page,
       },
       [QueryableField.LIMIT],
```

Comment:
We need to specify a default ordering. Can you add
`{
        orderBy: [[FillColumns.eventId, Ordering.ASC]],
      },`?

Reply:
Done https://github.com/dydxprotocol/v4-chain/pull/1420/commits/2fd8d09f2b96b3cb0e71f79b02c10d7b8bba992e#diff-84a6e6fd70f726b166899c1c25ded7304b24411adae6b71622815379318f6d3dR96